src/fcc_bridge_node_ros.cpp

```diff
@@ -0,0 +1,225 @@
+//
+// Created by Johan <job8197@thi.de> on 30.04.2024.
+//
+
+#include "fcc_bridge_node.hpp"
+
+// Libc header
+#include <chrono>
+
+namespace fcc_bridge {
+FCCBridgeNode::FCCBridgeNode(const std::string &name/*,
+                             const rclcpp::NodeOptions &node_options*/)
+    : CommonNode(name), internal_state(INTERNAL_STATE::STARTING_UP) {
+    // Pre-populate the last mission control heartbeat
+    this->last_mission_control_heartbeat.time_stamp = this->now();

```

Comment:
This look dangerous too me.
Mission Control might start later than the fcc bridge and the first heartbeat you will get can be older than your startup time + time_diff.
Or am I getting something wrong here?

Reply:
I assumed they would be started by ros launch and they should all start publishing within 500ms.
If this poses an issue in the simulation I can of cause change the behaivior.

Reply:
Alright